modal/_output.py

```diff
@@ -446,12 +447,14 @@ async def _get_logs():
                 pass
             elif log_batch.pty_exec_id:
                 if pty_shell_finish_event:
-                    print("ERROR: concurrent PTY shells are not supported.")
+                    logger.error("ERROR: concurrent PTY shells are not supported.")
                 else:
                     output_mgr.hide_status_spinner()
                     output_mgr._visible_progress = False
                     pty_shell_finish_event = asyncio.Event()
-                    asyncio.create_task(stream_pty_shell_input(client, log_batch.pty_exec_id, pty_shell_finish_event))
+                    task_context.create_task(
```

Comment:
Noticed we had this dangling asyncio task which could randomly get cancelled when it gets garbage collected, so added a TaskContext to give it an explicit life time 😬